tests/alfacase/test_case_description.py

```diff
@@ -888,6 +888,45 @@ def test_pvt_model_table_parameters_description_create_constant():
     np.array_equal(pvt.table_variables[1], expected_value.flatten())
 
 
+def test_pvt_model_table_parameters_description_create_constant_ph_table():
+    import numpy as np
+
+    pressure_values = np.linspace(0.5, 1e10, 4)
+    enthalpy_values = np.linspace(2617360.0, 2869860, 30)
+    h, p = np.meshgrid(enthalpy_values, pressure_values)
+    r = 286.9  # Air individual gas constant [J/kg K]
+
+    # Check constant_gas_density_model
+    rho_g_ref = 42.0
+    gas_density_expected_value = rho_g_ref + 0 * p
+
+    cp_g_ref = 1010.0
+    h_l_ref = 104.86e3
+
+    pvt = case_description.PvtModelTableParametersDescription.create_constant_ph_table(
+        ideal_gas=False,
+        rho_g_ref=rho_g_ref,
+        cp_g_ref=cp_g_ref,
+        h_l_ref=h_l_ref,
+        has_water=True,
+    )
+
+    t = pvt.table_variables[-1]
+
+    # Check gas_density_derivative_respect_pressure
+    gas_density_derivative_expected_value = 1 / (r * t)
+
+    h_lg = 2.260e6
+    temperature_expected_value = (h - h_lg - h_l_ref) / cp_g_ref
+
+    assert np.array_equal(pvt.table_variables[0], gas_density_expected_value.flatten())
```

Comment:
Consider using `pytest.approx` here:

```suggestion
    from pytest import approx
    assert pvt.table_variables[0] == approx(gas_density_expected_values)
```